### ISSUE TITLE
Error with vue-template-validator and empty attributes

### ISSUE THREAD

**Issue by carlosingles** [vuejs/vue-loader#130]
I'm having problems with `vue-loader@8.0.2` and compiling my *.vue files, error below:

```
Module build failed: TypeError: Cannot read property 'startOffset' of undefined
    at ...../node_modules/vue-loader/node_modules/vue-template-validator/lib/rules/camel-case-attr.js:25:37
```

I investigated my code and found that I was getting this error on any .vue file that had an empty attribute (including directives)

The ones I had issues with were `v-el:xxx` (directive) and `multiple` (for `<select>`)

I was able to fix this by ensuring there was a value for empty attributes, as in:

`<form v-el:form="">`

and

`<select multiple="multiple">`

I assume this isn't intended behaviour since the docs for Vue state that `v-el` does not expect an expression.

**Comment by wusisu** [comment]
+1 same problem.  

## error recur
can not build [vue-strap](https://github.com/yuche/vue-strap) error in `vue-strap/src/Select.vue`  
with `vue template`: 
```html
...
<button v-el:btn ...
...
```
error occurred in `vue-template-validator/lib/rules/camel-case-attr.js 20:37` which is "TypeError: Cannot read property 'startOffset' of undefined"  

## debug message
in `camel-case-attr.js`
>var name = attr.name
      // if the parsed attr still has camelCase name, then
      // it is a valid camelCase attribute, e.g. on SVG elements
      // see vue-loader#127
      if (hyphenateRE.test(name)) {
        return
      }
      var loc = node.__location.attrs[name]

`name` : `v-el:btn`
`node.__location.attrs` : `{ type: [Object],class: [Object],'@click': [Object],'@blur': [Object] }`

**Comment by yyx990803** [comment]
This is a bug in `parse5` (introduced by me, sadly) which I will submit a PR for.

**Comment by yyx990803** [comment]
For now you can modify `vue-template-validator/lib/rules/index.js` and comment out the `camel-case-attr` rule.

**Comment by yyx990803** [comment]
Update: should be fixed once https://github.com/inikulin/parse5/pull/95 gets merged and a new patch version of `parse5` is out.

**Comment by yyx990803** [comment]
Now fixed in `parse5@2.1.1`.
